### PR TITLE
ENH: Update http to https links where valid

### PR DESCRIPTION
**Pull request by tbirdso**
Automated reformatting with https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/Maintenance/ApplyScriptToRemotes.sh

Note that this also adds Python 3.10 wheels as a side effect of verifying ITK Python wheels are at 5.3rc04. If preferred this change could be removed from this PR.

### CODE REVIEW

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -200,7 +200,7 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        python-version: [37, 38, 39]
+        python-version: [37, 38, 39, 310]
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml]: Maybe don't add 3.10 to the list now. Or even better, fix the issue with Python 3.10 package generation 😄

**Review by dzenanz — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test-package.yml (hunk 1 above)
Comment (tbirdso) [reply .github/workflows/build-test-package.yml]: Python 3.10 CI fails at the SlicerExecutionModel build stage, perhaps the issue lies there?

Comment (dzenanz) [reply .github/workflows/build-test-package.yml]: Then don't add 3.10 to the list now. Somebody else will deal with that later.

Comment (tbirdso) [reply .github/workflows/build-test-package.yml]: Removed.

**Review by dzenanz — APPROVED** [review]

**Pull request merged**